lib/general.js

```diff
@@ -119,6 +119,39 @@ class OptionalFfmpegCommandCheck extends DoctorCheck {
 }
 checks.push(new OptionalFfmpegCommandCheck());
 
+
+class OptionalMjpegConsumerCommandCheck extends DoctorCheck {
+  async diagnose () {
+    let stdout = '';
+    const packageName = 'mjpeg-consumer';
+
+    const npmPath = await resolveExecutablePath(`npm${system.isWindows() ? `.cmd` : ''}`);
+    if (!npmPath) {
+      return nokOptional(`'npm' binary not found in PATH: ${process.env.PATH}`);
+    }
+
+    try {
+      ({stdout} = await exec(npmPath, ['list', '-g', packageName]));
+    } catch (err) {
+      log.debug(err);
+    }
+    if (stdout.includes(packageName)) {
+      const lines = stdout.split(EOL);
+      const mjpegConsumerLib = lines.find(function (line) { return line.includes(`${packageName}@`); });
```

Comment:
This can just be 
```js
const mjpegConsumerLib = stdout.split(EOL).find((line) =>  line.includes(`${packageName}@`));
```
Alternatively, using `npm ls -g -j mjpeg-consumer` you can parse it as JSON and just pull the version out:
```
$ npm ls -g -j mjpeg-consumer
{
  "dependencies": {
    "mjpeg-consumer": {
      "version": "1.1.0",
      "from": "mjpeg-consumer",
      "resolved": "https://registry.npmjs.org/mjpeg-consumer/-/mjpeg-consumer-1.1.0.tgz"
    }
  }
}

$ npm ls -g -j something-non-existent
{}
```

Reply:
great... I did not notice the `-j` option...

Reply:
I'll change the logic to use `-j` in https://github.com/appium/appium-doctor/issues/65